eq_translations/survey_schema.py

```diff
@@ -17,7 +17,8 @@ class SurveySchema:
         'playback',
         'item_title',
         'add_link_text',
-        'empty_list_text'
+        'empty_list_text',
+        'instruction'
```

Comment:
```suggestion
        'instruction',
```

Reply:
Leaving a trailing comma in these lists allows for cleaner diffs when elements are added

Reply:
Sorted, thanks!